Server/mods/deathmatch/logic/luadefs/CLuaDefs.cpp

```diff
@@ -113,11 +114,9 @@ int CLuaDefs::CanUseFunction(lua_CFunction f, lua_State* luaVM)
     {
         return true;
     }
-
+    
     // Get associated resource
-    CResource* pResource = m_pResourceManager->GetResourceFromLuaState(luaVM);
-    if (!pResource)
-        return true;
+    auto pResource{ lua_getownerresource(luaVM) };
```

Comment:
```suggestion
    CResource* pResource{ lua_getownerresource(luaVM) };
```
IMHO, `auto` only makes sense if you can see the type in the assignment